integration_tests/mockApis/prison.ts

```diff
@@ -1206,7 +1206,7 @@ export default {
         headers: {
           'Content-Type': 'application/json;charset=UTF-8',
         },
-        jsonBody: [{ number: '07700000000', type: 'mobile' }],
+        jsonBody: personId === 9876543 ? [] : [{ number: '07700000000', type: 'mobile' }],
       },
     })
   },
```

Comment:
These seem incredibly specific. Can we not just have the required response as an argument with a default?

Reply:
Yes, that works better. Thanks.